include/tbl/cli_cmds.h

```diff
@@ -149,7 +149,7 @@ CLI_CMD(VCL_SHOW,
 	"vcl.show [-v] <configname>",
 	"Display the source code for the specified configuration.",
 	"",
-	1, 2
+	0, 2
```

Comment:
@Dridi on second thoughts I must disagree here, simply because the convention appears to be to use angled brackets (see other command syntax descriptions), my change here is to simply allow for 0 parameters to be provided.